src/app/core/entity-components/entity-utils/dynamic-form-components/edit-month/edit-month.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, ViewEncapsulation } from "@angular/core";
+import { CommonModule } from "@angular/common";
+import { EditComponent } from "../edit-component";
+import { MatFormFieldModule } from "@angular/material/form-field";
+import { MatInputModule } from "@angular/material/input";
+import {
+  MatDatepicker,
+  MatDatepickerModule,
+} from "@angular/material/datepicker";
+import { ReactiveFormsModule } from "@angular/forms";
+import { ErrorHintComponent } from "../../error-hint/error-hint.component";
+import {
+  DateAdapter,
+  MAT_DATE_FORMATS,
+  MAT_DATE_LOCALE,
+} from "@angular/material/core";
+import { Moment } from "moment";
+import {
+  MAT_MOMENT_DATE_ADAPTER_OPTIONS,
+  MomentDateAdapter,
+} from "@angular/material-moment-adapter";
+
+export const MY_FORMATS = {
+  parse: {
+    dateInput: "YYYY-MM",
+  },
+  display: {
+    dateInput: "YYYY-MM",
+    monthYearLabel: "YYYY MMM",
+    dateA11yLabel: "LL",
+    monthYearA11yLabel: "YYYY MMMM",
+  },
+};
+
+@Component({
+  selector: "app-edit-month",
+  standalone: true,
+  imports: [
+    CommonModule,
+    MatFormFieldModule,
+    MatInputModule,
+    MatDatepickerModule,
+    ReactiveFormsModule,
+    ErrorHintComponent,
+  ],
+  templateUrl: "./edit-month.component.html",
+  styleUrls: ["./edit-month.component.scss"],
+  providers: [
+    { provide: MAT_DATE_FORMATS, useValue: MY_FORMATS },
+    {
+      provide: DateAdapter,
+      useClass: MomentDateAdapter,
```

Comment:
is this only possible using MomentDateAdapter? We usually didn't use this and have been discussing to remove moment, as it is not maintained anymore ...

Reply:
Unfortunately, the default adapter does not support parsing/formatting dates (see [docs](https://material.angular.io/components/datepicker/overview#choosing-a-date-implementation-and-date-format-settings))